packages/moonstone/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 The following is a curated list of changes in the Enact moonstone module, newest changes on the top.
 
+## unreleased
+
+### Added
+
+- `moonstone/Header` prop `hideLine` to hide `border-bottom`
```

Comment:
This is a bit too implementation-specific.  I'll fix it.

Reply:
Also, there is no `moonstone/Header` component.

Reply:
Nice change (collection of changes), @webOS101!